libnmstate/nm/route.py

```diff
@@ -63,14 +64,18 @@ def get_config(acs_and_ip_profiles):
     """
     routes = []
     for (active_connection, ip_profile) in acs_and_ip_profiles:
-        if not ip_profile.props.routes:
+        if not ip_profile.props.routes and not ip_profile.props.gateway:
```

Comment:
You should probably fetch both values into variables as they are re-used all the time later on.